suite-native/discovery/src/discoveryConfigSlice.ts

```diff
@@ -0,0 +1,29 @@
+import { createSlice, PayloadAction } from '@reduxjs/toolkit';
+
+export interface DiscoveryConfigState {
+    areTestnetsEnabled: boolean;
+}
+
+export type DiscoveryConfigSliceRootState = {
+    wallet: { discoveryConfig: DiscoveryConfigState };
+};
+
+export const discoveryConfigInitialState: DiscoveryConfigState = {
+    areTestnetsEnabled: false,
+};
+
+export const discoveryConfigSlice = createSlice({
+    name: 'discoveryConfig',
+    initialState: discoveryConfigInitialState,
+    reducers: {
+        setAreTestnetsEnabled: (state, { payload }: PayloadAction<boolean>) => {
```

Comment:
nit: toggleAreTestnetsEnabled (https://redux.js.org/style-guide/#model-actions-as-events-not-setters)

Reply:
(No need for payload)

Reply:
changed in 4a8d956e6d